### DISCUSSION TITLE
Koinly CSV Export Not Valid

### DISCUSSION THREAD

**Discussion by WolfPG** [pwillworth/dfkreport#32] [Q&A] [answered]
Hi,

I'm exporting my report as a csv file for Koinly. When I import it to Koinly it states "Unknown File - Unable to import this file automatically. Proceed to the file mapping page to continue the import process."

Is there an update required to ensure the CSV export for Koinly works correctly?

**Comment by WolfPG** [comment]
Seems the headings are incorrect? This is what Koinly states is the format required for Koinly (Universal) CSV imports - https://docs.google.com/spreadsheets/d/1dESkilY70aLlo18P3wqXR_PX1svNyAbkYiAk2tBPJng/edit#gid=0

**Reply by WolfPG** [reply, marked as answer]
Yes, that's worked now. So the CSV export doesn't use the correct headings. I updated the headings with the following and then the import worked:

- Date
- Sent Amount
- Sent Currency
- Received Amount
- Received Currency
- Fee Amount
- Fee Currency
- Net Worth Amount
- Net Worth Currency
- Label
- Description
- TxHash

**Reply by pwillworth** [reply]
Thanks for the info, they must have updated the format since I implemented.  I'll get that updated soon so it exports the right headers without having to modify.
